src/app.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+const { generateNumber } = require('./modules/generateNumber');
+const { makeTry } = require('./modules/makeTry');
+
+const winCombination = generateNumber();
+
+// eslint-disable-next-line no-console
+console.log('Hi! This is Bulls & Cows game.');
```

Comment:
Avoid using 'console.log' for user interaction. Consider creating a separate module for input/output operations as specified in the task description.